static/skywire-manager-src/src/app/services/vpn-client-discovery.service.ts

```diff
@@ -0,0 +1,141 @@
+import { Injectable } from '@angular/core';
+import { Observable, of } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { retryWhen, delay, map } from 'rxjs/operators';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skywire/pullRequest?prid=7163595)
```suggestion
import { retryWhen, delay, map } from "rxjs/operators";
```